pkg/cri/instrument/instrumented_service.go

```diff
@@ -90,7 +98,14 @@ func (in *instrumentedService) checkInitialized() error {
 // GRPC service request handlers should return error before server is fully
 // initialized.
 // NOTE(random-liu): All following functions MUST check initialized at the beginning.
-func (in *instrumentedAlphaService) checkInitialized() error {
+func (in *instrumentedAlphaService) checkInitialized(ctx context.Context) error {
+	log.G(ctx).Warning("CRI API v1alpha2 is deprecated since containerd v1.7 and removed in containerd v2.0. Use CRI API v1 instead.")
```

Comment:
Move this inside the `emitWarnign.Do` as well so we don't fill the log with a bunch of repeated warnings every time an RPC is invoked.

Reply:
Done